src/components/home/dashboard/EntityStatOverview/useActiveEntityCount.ts

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from '@supabase-cache-helpers/postgrest-swr';
+import { getLiveSpecShards } from 'api/liveSpecsExt';
+import { useMemo } from 'react';
+import { useTenantStore } from 'stores/Tenant/Store';
+import { Entity } from 'types';
+import { hasLength } from 'utils/misc-utils';
+
+export default function useActiveEntityCount(entityType: Entity) {
+    const selectedTenant = useTenantStore((state) => state.selectedTenant);
+
+    const { data, error, isLoading } = useQuery(
+        hasLength(selectedTenant)
+            ? getLiveSpecShards(selectedTenant, entityType)
+            : null
+    );
```

Comment:
Forgot... since this is fetching an untold number of live specs we should probably update the UI to have a UX for `1000+` active entities and then later on decide if we ever want to show more than that in the dashboard.